patchwork-fml/src/main/java/net/minecraftforge/fml/javafmlmod/FMLModContainer.java

```diff
@@ -47,4 +48,18 @@ public IEventBus getEventBus() {
 	private void onEventFailed(IEventBus iEventBus, Event event, IEventListener[] listeners, int i, Throwable throwable) {
 		// TODO
 	}
+
+	@Override
+	protected void acceptEvent(final Event e) {
+		this.eventBus.post(e);
+	}
+
+	@Override
+	public Object getMod() {
+		return this.instance;
```

Comment:
this seems to always be null, it might actually be better to drop them until the setter is actually called

Reply:
Should we just comment getMod() and setMod() for now? Impl setMod needs some changes in the patcher as well, the ForgeInitializer.onForgeInitialize should return the instance of the Forge mod class, so that we can record it in its ModContainer.

Reply:
yeah, just comment them out for now and the change that needs Patcher can be a separate PR

Reply:
Can we leave getMod() and setMod() here? Just add TODOs to both of them and throw a warning when they are called? Other methods in this pr require getMod(), simply remove getMod() needs more work.

Reply:
Alright. Make sure you open an issue on Patcher about it